### ISSUE TITLE
Traits not inheriting resourcePathName/resourcePath values in api console - RAML 1.0

### ISSUE THREAD

**Issue by impatient** [mulesoft/api-workbench#155]
When you have a path with a resource type + parameter that calls a trait with the parameter, the values of resourcePathName, resourcePath and parameter name are not populated when the trait is evaluated.

Sample project here:  https://github.com/impatient/raml-inheritance-sample

In the api.raml we have:

```
/alpha:
  /letters:
    type: { collection: { parentType: alpha } }
```  

In the collection resourceType, we have:

```

uses:
  defaultResponses: !include ../libraries/responseBase.raml
  error-types: !include ../types/error-types.raml

get:
  is: [defaultResponses.general: { parentType: <<parentType>> }]

```

In the api console, for the GET of /alpha/letters, we are missing the replaced values:

![image](https://cloud.githubusercontent.com/assets/3408099/14619068/47f1573e-057b-11e6-8e07-d62cc7104b3d.png)

**Comment by ddenisenko** [comment]
[raml-inheritance-sample-master 2.zip](https://github.com/mulesoft/api-workbench/files/481632/raml-inheritance-sample-master.2.zip)

I modified the original project to comply with the latest spec, and now workbench explodes when opening main raml file.

**Comment by ddenisenko** [comment]
This ticket is most probably parser-related. Posted two tickets into the parser repository: https://github.com/raml-org/raml-js-parser-2/issues/483 and https://github.com/raml-org/raml-js-parser-2/issues/484

**Comment by ddenisenko** [comment]
Lowering priority as the critical part is fixed until the other part is investigated.

**Comment by KonstantinSviridov** [comment]
The fix is going to be released in 0.8.39
